R/lastdose.R

```diff
@@ -113,40 +136,38 @@ lastdose <- function(data,..., include_ldos = TRUE, include_tafd = TRUE) {
 #' @rdname lastdose
 #' @export
 lastdose_list <- function(data,
-                          time_col = "TIME",
-                          time_units = NULL,
-                          id_col = "ID",
+                          time_col = find_time_col(data),
+                          time_units = getOption("lastdose.time_units", NULL),
+                          id_col = find_id_col(data),
                           fill = -99,
                           back_calc = TRUE,
                           addl_ties = c("obs_first", "dose_first"),
                           comments = find_comments(data)) {
-  if(length(comments)==1) {
+
+  if(length(comments) == 1) {
     comments <- rep(comments,nrow(data))
   }
-  if(!length(comments)==nrow(data)) {
-    stop("'comments' must be have length equal to the number of rows in 'data'",call.=FALSE)
+  if(!length(comments) == nrow(data)) {
```

Comment:
Is this:
```
  if(!length(comments) == nrow(data)) {
```
identical to:
```
  if(length(comments) != nrow(data)) {
```

When I run `!length(comments)` by itself I get `FALSE`, so wanted to confirm the functionality.

Reply:
Seemed to work as designed, just hadn't seen that syntax before
![image](https://user-images.githubusercontent.com/18685116/118017230-5d461280-b324-11eb-941b-d2fc675d1932.png)